Gemfile

```diff
@@ -50,13 +50,16 @@ gem "httpclient"
 gem "view_component", require: "view_component/engine"
 
 # Database based asynchronous priority queue system
+gem "daemons"
 gem "delayed_cron_job"
 gem "delayed_job_active_record"
 
 platform :mswin, :mingw, :x64_mingw do
   gem "wdm", ">= 0.1.0"
 end
 
+gem "addressable"
```

Comment:
Just wondering what we are using this for?

Reply:
It's used in the [school import](https://github.com/DFE-Digital/early-careers-framework/blob/5f7509105b5b88e503e5d3d92cfd6a33fc294648/app/services/school_data_importer.rb#L58). I have no idea why it stopped working, but when I was trying to run the import it complained about the lack of Addressable. Perhaps we removed something that had it as a transient dependency?